cmd/deployment-check/main.go

```diff
@@ -272,3 +274,15 @@ func reportToKuberhealthy(ok bool, errs []string) {
 	}
 	return
 }
+
+// ReportFailureAndExit logs and reports an error to kuberhealthy and then exits the program.
+// If a error occurs when reporting to kuberhealthy, the program fatals.
+func ReportFailureAndExit(err error) {
```

Comment:
what is the need of this func?

Reply:
I was fatl and reporting to kuberhealthy in the event of an error in multitolerations...however I can handle the error instead and set a default toleration and remove this func